ca/ca.go

```diff
@@ -208,30 +208,22 @@ func makeCertificateProfilesMap(defaultName string, profiles map[string]issuance
 		}
 		hash := sha256.Sum256(encodedProfile.Bytes())
 
-		_, ok := profileByName[name]
-		if !ok {
-			profileByName[name] = &certProfileWithID{
-				name:    name,
-				hash:    hash,
-				profile: profile,
-			}
-		} else {
-			return certProfilesMaps{}, fmt.Errorf("duplicate certificate profile name %s", name)
+		cpwid := certProfileWithID{
```

Comment:
I'd like to do less of the pattern where we name a variable using an initialism of the type name. It leads to confusing variable names: long enough to be potentially meaningful, but not actually meaningful.

How about "withID"? Or even "profile2", since all we're doing here is adding some data to "profile". If we could shadow it with a different type I'd suggest that but we can't do that within a single scope.

Reply:
Good point, I like `withID`.